Kconfig

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2023 Renesas Electronics Corporation.
+# Copyright (C) 2023 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+config DOMD_ID
```

Comment:
We should use some prefix for all config items to distinguish between other config options. Should be: `CONFIG_AOS_..."

Reply:
done